packages/component/src/Styles/StyleSet/SuggestedAction.js

```diff
@@ -29,7 +37,11 @@ export default function createSuggestedActionStyle({
         borderRadius: suggestedActionBorderRadius,
         fontFamily: primaryFont,
         fontSize: 'inherit',
-        height: suggestedActionHeight,
+        height: suggestedActionsStackedLayoutButtonTextWrap ? 'auto' : suggestedActionHeight,
+        maxHeight: suggestedActionsStackedLayoutButtonTextWrap
```

Comment:
Use CSS BEM instead of tertiary operators.

For example, by applying modifiers `webchat__suggested-action--wrapping`.